src/App.tsx

```diff
@@ -1,14 +1,23 @@
-import { useState } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+import FullScreenPage from '../src/components/FullScreenPage'
 
 
-function App() {
-  const [count, setCount] = useState(0)
+const App: React.FC=() =>{
+  
 
   return (
-    <>
-      
-    </>
+    <Wrapper>
+      <FullScreenPage children/>
```

Comment:
remove these components from here. 
We will add it when we gonna work with pages.